spec/policies/application_policy_spec.rb

```diff
@@ -21,9 +21,9 @@
     end
   end
 
-  context 'the person is a pleb' do
-    subject { ApplicationPolicy.new(pleb, 'any object') }
-    let(:pleb) { FactoryGirl.build(:person) }
```

Comment:
Could technically be squashed in the git history so that correct language is always used in our history